README.md

```diff
@@ -13,9 +8,15 @@ and [RDS Enhanced Monitoring via CloudWatch Logs](https://docs.aws.amazon.com/Am
 Based on [Technofy/cloudwatch_exporter](https://github.com/Technofy/cloudwatch_exporter),
 but very little of the original code remained.
 
+Forked from [Percona RDS exporter](https://github.com/percona/rds_exporter). **Enhancements:**
+* Dockerfile was rewritten to multistage build
+* Added script `generate_rds_list.py` that creates list of RDS instances.  
+Script supports filtering: if `USE_FILTER` environment variable was set with `true` value, only metrics for instances with tag `UseExporter=true` will be listed. 
+* Supervisord feature for Docker. RDS_exporter itself does not support configuration file rereading - in Docker it is launched via supervisord.  
+Every 20 minutes configuration file is regenerated and rds_exporter is restarted (via cron job).
```

Comment:
Every 20 minutes the configuration file is compiled and rds_exporter is eventually restarted (via cron job).

Reply:
Fixed in https://github.com/pdffillerdocker/rds_exporter/commit/4e36931a0807b3e836b4f63cea3352a3ee36ed68